tests/distributed/test_distributed_sampling.py

```diff
@@ -132,10 +132,69 @@ def test_rpc_sampling_shuffle():
         check_rpc_sampling_shuffle(Path(tmpdirname), 2)
         check_rpc_sampling_shuffle(Path(tmpdirname), 1)
 
+def start_in_subgraph_client(rank, tmpdir, disable_shared_mem, nodes):
+    import dgl
+    gpb = None
+    if disable_shared_mem:
+        _, _, _, gpb = load_partition(tmpdir / 'test_in_subgraph.json', rank)
+    dist_graph = DistGraph("rpc_sampling_ip_config.txt", "test_in_subgraph", gpb=gpb)
```

Comment:
change "rpc_sampling_ip_config.txt" And I would prefer put them in separate files for simplicity